src/validation/rules/UniqueDirectivesPerLocation.js

```diff
@@ -48,7 +70,11 @@ export function UniqueDirectivesPerLocation(
               ]),
             );
           } else {
-            knownDirectives[directiveName] = directive;
+            const repeatable = repeatableMap[directiveName];
+
+            if (repeatable === undefined || !repeatable) {
```

Comment:
Up to discussion: I don't think unknown directives should be treated as non-repeatable. Unknown directives should trigger `KnownDirectives` and other rules shouldn't make any assumption about it.
Previously it was safe since all directives were non-repeatable.

Reply:
It is a good point. After thinking about it, I think I would agree with you. In both scenations: directive is not known and it is known but repeatable, it should be excluded from this validation entirely. I will adjust the logic.